confluence/models/content.py

```diff
@@ -60,7 +60,12 @@ class Content:
     """
 
     def __init__(self, json):  # type: (Dict[str, Any]) -> None
-        self.id = json['id']  # type: int
+        # attachment id get returned starting with att which can be stripped
+        # this ensures ids are always of type int
+        id = str(json['id'])
```

Comment:
Quite sad about this :/. Not your fault, I think I should have just used str representation of ids in the first place but it's too late to change that now and risk breaking downstream apps.

Reply:
Yes it is annoying, I think this is slightly better than going through and updating all the id types to be Union[int, str] but happy to go that way too.

Reply:
No, agreed. Nothing to change here, just annoyed with myself :)